cmake/modules/SwiftSupport.cmake

```diff
@@ -0,0 +1,105 @@
+#[[
```

Comment:
Yeah, its replicated everywhere because we don't have a good home for the module. I would like to obsolete it by adding support for installing Swift modules to CMake, but it is the current state. I agree with you though that it is a lot to keep in sync - I've felt the pain as I did the ARM64 fixes for Windows.